lib/Core/Executor.cpp

```diff
@@ -117,6 +117,9 @@ cl::OptionCategory TestGenCat("Test generation options",
 
 namespace {
 
+cl::OptionCategory TestCaseCat("Test case options",
```

Comment:
What's this?

Reply:
This is the category to which the `--start-fn` and `--end-fn` cmd line args belong. I should have put them in `namespace klee`. Adding a commit for this as well.